src/displayapp/widgets/Counter.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#include <lvgl/lvgl.h>
+
+namespace Pinetime {
+  namespace Applications {
+    namespace Widgets {
+      class Counter {
+      public:
+        Counter(int min, int max);
+
+        void Create();
+        static void upBtnEventHandler(lv_obj_t* obj, lv_event_t event);
+        static void downBtnEventHandler(lv_obj_t* obj, lv_event_t event);
+        void Increment();
+        void Decrement();
+        void SetValue(int newValue);
+
+        int GetValue() const {
+          return value;
+        }
+
+        lv_obj_t* GetObject() const {
```

Comment:
I'd move the body of the function into the cpp file

Reply:
small member functions can get easily inlined by the compiler. Thanks @Riksu9000 for pointing that out